code/modules/mob/living/simple_animal/hostile/megafauna/blood_drunk_miner.dm

```diff
@@ -39,8 +39,8 @@ Difficulty: Medium
 	ranged = TRUE
 	ranged_cooldown_time = 16
 	pixel_x = -16
-	crusher_loot = list(/obj/structure/closet/crate/necropolis/bdm, /obj/item/crusher_trophy/miner_eye)
-	loot = list(/obj/structure/closet/crate/necropolis/bdm)
+	crusher_loot = list(/obj/structure/closet/crate/necropolis/bdm, /obj/item/disk/design_disk/modkit_disc/resonator_blast, /obj/item/disk/design_disk/modkit_disc/rapid_repeater, /obj/item/disk/design_disk/modkit_disc/mob_and_turf_aoe, /obj/item/disk/design_disk/modkit_disc/bounty, /obj/item/crusher_trophy/miner_eye)
+	loot = list(/obj/structure/closet/crate/necropolis/bdm, /obj/item/disk/design_disk/modkit_disc/resonator_blast, /obj/item/disk/design_disk/modkit_disc/rapid_repeater, /obj/item/disk/design_disk/modkit_disc/mob_and_turf_aoe, /obj/item/disk/design_disk/modkit_disc/bounty)
```

Comment:
Move the discs to the chest he drops instead, but have it only choose 1-2 rather than giving all 5 at once.

Reply:
Sounds good

Reply:
It will now give 1 random disk inside of the chest